### PR TITLE
Region & topic modeling

### PR DESCRIPTION
**Pull request by davidfischer**
* Adds regions and topics to our database
* There's a data migration to preload the regions and topics we use
* The Flight edit and display UIs show targeted regions and topics
* There's a local memory cache for regions and topics to avoid hitting them during ad decisions

## Screenshot
Screenshot from 2022-05-23 10-50-04

## Not in this PR

* No changes were made to how the aggregations and reports around regions and topics were made yet
* The analyzer does not yet work off these keywords

### CODE REVIEW

**Review by ericholscher — APPROVED** [review]
This looks great! Excited to get this out.

File: adserver/models.py [hunk 1]
```diff
@@ -670,6 +823,20 @@ def show_to_geo(self, geo_data):
         if self.excluded_countries and geo_data.country in self.excluded_countries:
             return False
 
+        # Check region groupings as well
+        if self.included_regions or self.excluded_regions:
+            # Only load regions if we have to
+            regions = Region.load()
```
[the remark this answers is not in the document]
Comment (davidfischer) [reply adserver/models.py:829]: `load_cache()`?

File: config/settings/base.py [hunk 2]
```diff
@@ -160,14 +160,20 @@
 
 # Caching
 # Using a local memory cache for development and testing
-# and a more hardened cache in production
+# although production typically uses Redis
 # See: https://docs.djangoproject.com/en/3.2/topics/cache/
 # --------------------------------------------------------------------------
+CACHE_LOCAL_ALIAS = "local"
```
[the remark this answers is not in the document]
Comment (davidfischer) [reply config/settings/base.py:166]: Yup, basically.

File: adserver/forms.py [hunk 3]
```diff
@@ -187,86 +201,12 @@ def __init__(self, *args, **kwargs):
...
             Fieldset(
                 _("Flight targeting"),
-                Div(
-                    "include_countries",
-                    Div(
-                        HTML(
-                            format_html(
-                                """
-                                <ul class="list-inline">
-                                    <li class="list-inline-item"><a class="ea-update-field" href="#" data-value="{}" data-target-field="#id_include_countries">US / Canada</a></li>
```
[the remark this answers is not in the document]
Comment (davidfischer) [reply adserver/forms.py:197]: Ya. More frequently people just want some subset where they will be enumerating countries. I'd say people adding a region and just excluding one or two isn't common.

File: adserver/models.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ericholscher) [reply adserver/models.py:829]: `load_from_cache`? This seems different than "loading", since we're also getting data, but seems fine.

**Comment by davidfischer** [comment]
The CI failure is docs related. I'll get a fix for it.

**Pull request merged**